src/functions/DidSiopMetadata.ts

```diff
@@ -31,16 +162,18 @@ function verifySubjectIdentifiers(subjectIdentifiers: SubjectIdentifierType | Su
   return subjectIdentifiers === SIOP.SubjectIdentifierType.DID;
 }
 
-function verifyDidMethods(methodsA: string[] | string, methodsB: string[] | string): void {
+function verifyDidMethods(methodsA: string[] | string, methodsB: string[] | string): Array<string> {
   const supportedDidMethods = getIntersection(methodsA, methodsB);
```

Comment:
Suggest to do the above check for empty rp metadata first in this method. If the RP supports everything, then simply return all the results passed in for the OP

Reply:
Fixed!